src/components/ImageGallery.jsx

```diff
@@ -762,14 +762,17 @@ class ImageGallery extends React.Component {
     return items.length >= 2;
   }
 
+  //SOLVED BY hunxjunedo : https://github.com/xiaolin/react-image-gallery/issues/767
+  //removed some minor errors, rmeove semi-colon
   canSlideLeft() {
-    const { infinite } = this.props;
-    return infinite || this.canSlidePrevious();
+    const { infinite, isRTL } = this.props;
+    //so basically is the list is right to left, the canSlideLeft depends on canSlideNext instead of canSlideRight
```

Comment:
Grammar fix here, maybe something simpler like:

`// swap the logic here when RTL is active`